docs/sitemap.md

```diff
@@ -0,0 +1,34 @@
+# Site map
+
+- Home Page
+  - sizzle
```

Comment:
Also, I think it would be helpful to differentiate proposed pages and their titles from concepts you want the parent page to include. Perhaps simply capitalize the former?